### ISSUE TITLE
Wrong format for custom file path

### ISSUE THREAD

**Issue by ganyunhee** [bingryan/obsidian-markdown-export-plugin#49]
Issue
---
Plugin collects images properly onto default `attachment` folder and refers to them in the form of `attachment/image_name.png`
But with a custom file path like `posts/posts_images`, it changes forward slash (`/`) to backslash (`\`) thereby creating incorrect image referrence.

Settings Preview:
---
<img width="828" alt="file_path" src="https://github.com/bingryan/obsidian-markdown-export-plugin/assets/81596122/6cc3c160-5eff-496a-a1d7-b44ecaeb5b00">

Created markdown file Preview:
---
<img width="766" alt="wrong_referral" src="https://github.com/bingryan/obsidian-markdown-export-plugin/assets/81596122/c5a01f64-95c9-4043-84d6-c59b3dcb54ca">

**Comment by bingryan** [comment]
@ganyunhee windows?

**Comment by ganyunhee** [comment]
Yes. Windows 11. Specifically, Education version 22H2. Sorry for the late reply. Let me know if you need more details.

**Comment by bingryan** [comment]
@ganyunhee If yours is windows, you should use the path below windows(`posts\posts_images`), BTW, you can use default value: `attachment`
